client/my-sites/plans/current-plan/current-plan-thank-you-card/free-plan-thank-you-card.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+import { localize } from 'i18n-calypso';
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+
+export class FreePlanThankYouCard extends Component {
+	render() {
+		const { siteSlug, translate } = this.props;
+		return (
+			<Card className="current-plan-thank-you-card__main">
+				<div className="current-plan-thank-you-card__content">
+					<img
+						className="current-plan-thank-you-card__illustration"
+						alt=""
+						aria-hidden="true"
+						src="/calypso/images/illustrations/security.svg"
+					/>
+					<h1 className="current-plan-thank-you-card__title">
+						{ translate( 'Welcome to Jetpack Free!' ) }
+					</h1>
+					<p>
+						{ translate( 'We’ve automatically begun to protect your site from attacks.' ) }
```

Comment:
This line uses a fancy `’`, while the next string uses `'`. Let's be consistent.